tests/audit-tests/run-tests.sh

```diff
@@ -3,7 +3,9 @@
 set -eu
 
 gradle -v
-#./gradlew clean test -Dcucumber.filter.tags="$environment"
+
+echo "Current Environment $TEST_ENVIRONMENT"
+#./gradlew clean test -Dcucumber.filter.tags="$TEST_ENVIRONMENT"
```

Comment:
```suggestion
#./gradlew clean test -Dcucumber.filter.tags="@$TEST_ENVIRONMENT"
```
I believe you need an @ to get the tagging to work